e2e/autoscript-apps/tsconfig.app.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "dist",
+    "strict": false,
```

Comment:
new typescript causes an error in sdk which blocks builds, we can fix it if we want but i didnt feel like i should include this in this branch as its SDK related not sample app related.

Reply:
Yeah, I had to add this to the Widget (`zod` requires it), and it did require some source code updates.